### ISSUE TITLE
Spread operator not supported in Node v4

### ISSUE THREAD

**Issue by jnovack** [Farof/heroprotocoljs#23]
I'm failing on v4.2.6 with this command, whatever it is. I've never seen it before.

```
[jnovack@gyarados ~/Source/heroprotocoljs]$ node bin/extract.js replays/towers-of-doom.StormReplay /tmp/ -p

/Users/jnovack/Source/heroprotocoljs/heroprotocol.js:137
            [...archive.protocol[decoderMap[archiveFile]](archive.readFile(archiveFile))];
             ^^^

SyntaxError: Unexpected token ...
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:374:25)
    at Object.Module._extensions..js (module.js:417:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/jnovack/Source/heroprotocoljs/bin/extract.js:13:22)
    at Module._compile (module.js:410:26)
    at Object.Module._extensions..js (module.js:417:10)
```

**Comment by jnovack** [comment]
Well, this works in v5.5.0.

Unfortunately, I believe we're going to have to support v4+, it is considered "Long Term Support" by nodejs

**Comment by Farof** [comment]
Resolved 62135a031ffebde85b5369851767dd2236bc2a99.
